.rustfmt.toml

```diff
@@ -1,18 +0,0 @@
-match_block_trailing_comma = true
-max_width = 120
```

Comment:
This is especially weird, this is a Mozilla library and still doesn't follow what mozilla-central does (the 80-char default width from clang-format) 🤔